src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,32 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class encapsulating the event task.
+ */
+public class Event extends Task {
+    private LocalDate from;
+    private LocalDate to;
+
+    /**
+     * Constructor for an Event
+     *
+     * @param description
+     * @param from
+     * @param to
```

Comment:
I believe the @params can be omitted since they are obvious in this context.
https://se-education.org/guides/conventions/java/intermediate.html#comments